GliaWidgetsTests/Sources/Glia/GliaTests.swift

```diff
@@ -603,6 +604,7 @@ final class GliaTests: XCTestCase {
         var engCoordEnvironment = EngagementCoordinator.Environment.engagementCoordEnvironmentWithKeyWindow
         engCoordEnvironment.fileManager = .mock
         environment.createRootCoordinator = { _, _, _, _, _, _, _ in EngagementCoordinator.mock(environment: engCoordEnvironment) }
+        environment.queuesMonitor = .mock
```

Comment:
It is not related to current test, but globally for all tests: `QueuesMonitor` is reference type, and it's `mock` instance  right now is shared across all unit tests, which may affect some test in very non-obvious ways, because internal state of the instance may be change by other tests. This is because `QueuesMonitor.mock` is implemented like this right now:
```
extension QueuesMonitor {
    static let mock = QueuesMonitor(environment: .mock) // same shared instance
}
```
For reference types is is suggested to used functions instead of constants:
```
extension QueuesMonitor {
    static func mock(environment: QueuesMonitor.Environment =  .mock) -> QueuesMonitor {
        QueuesMonitor(environment: environment)
    }
}
```

This will guarantee that new instance of `QueuesMonitor` is created for every test. Also using `func` makes this more obvious compared to computed property (which can also return new instance). In addition func can accepts params with defaultzmock values which is handy to pass when needed.

For structs though using `static let mock = Struct()` is fine, because struct is copied automatically, thus not sharing internal state.